src/lib/registration/ui/addon_selection_base_dialog.rb

```diff
@@ -96,7 +96,7 @@ def content
           Left(CheckBox(Id(:filter_beta), Opt(:notify),
             _("&Filter Out Beta Versions"), FILTER_BETAS_INITIALLY)),
           addons_box,
-          Left(Label(_("Details"))),
+          Left(Label(_("Details (english only)"))),
```

Comment:
Please use capital `E`, the language names always use a capital letter in English.

And please add a `.changes` entry with the `bsc` reference and bump the version to `3.2.6` in the spec file (manually or run `rake version:bump`).

Thank you!